CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -1,5 +1,22 @@
 # Changelog 1.15
 
+## 1.15.1 - 2024-03-18
+
+### Changed
+
+- Stop using `projects.registry.vmware.com` for user-facing images. ([#6073](https://github.com/antrea-io/antrea/pull/6073), [@antoninbas])
+- Persist TLS certificate and key of antrea-controller and periodically sync the CA cert to improve robustness. ([#5955](https://github.com/antrea-io/antrea/pull/5955), [@tnqn])
```

Comment:
For v1.15.1, we also backported the PR5998 to disable cgo,  I plan to update the log and add an item in the `Changed` with following line, please help to check if this works or more details needed @antoninbas 
```
- Disable cgo for all Antrea binaries. ([#5988](https://github.com/antrea-io/antrea/pull/5988), [@antoninbas])
```
For now, only windows containerd e2e failed, it will be fixed by this PR https://github.com/antrea-io/antrea/pull/6132.

Reply:
looks fine